main.tf

```diff
@@ -36,7 +36,7 @@ locals {
         ["bad location_type"] # will force an error
   )))
 
-  predefined_node_resource_labels = { TF_used_for = "gke", TF_used_by = google_container_cluster.k8s_cluster.name }
+  predefined_node_resource_labels = { tf_used_for = "gke", tf_used_by = google_container_cluster.k8s_cluster.name }
```

Comment:
Let’s use kebab case naming convention.

Same as what Google uses here
https://cloud.google.com/kubernetes-engine/docs/how-to/creating-managing-labels#automatically-applied-labels
![IMG_6724](https://github.com/user-attachments/assets/353dcbd5-18f0-4094-afcd-6cc503b11929)